src/server/index.js

```diff
@@ -14,12 +14,9 @@ const mongoose = require('mongoose');
 
 
 // SSR method importing from ./controller/ssr
-import { renderSSRcomponent } from "./controller/ssr";
```

Comment:
That import/require issue was here too ?

Reply:
yup this and the next line as well. Only these two places.

Reply:
Okay then 
This looks good then 😃